simcon_project/conversation_templates/models/template_node_response.py

```diff
@@ -5,6 +5,10 @@
 class TemplateNodeResponse (models.Model):
     id = models.UUIDField(unique=True, editable=False, primary_key=True, default=uuid.uuid4)
     transcription = models.CharField(max_length=1000)
-    template_node = models.ForeignKey('conversation_templates.TemplateNode',default=0, related_name='responses', on_delete=models.DO_NOTHING)
-    parent_template_response = models.ForeignKey('conversation_templates.TemplateResponse',default=0, related_name='node_responses', on_delete=models.CASCADE)
+    template_node = models.ForeignKey('conversation_templates.TemplateNode',default=0, related_name='responses',
```

Comment:
formatting?

Reply:
pycharm was yelling at me that they were too long

Reply:
It's a hard call with this long line restriction thing. I'd leave it as it was since it wasn't bad (template_node especially)... parent_template_response is getting close to when you should make things multi-line but I think its still good. Things should be made multi-line when you really need to scroll to understand what's going on... though it is a matter of preference tbh.